docker-compose.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo docker-compose up --scale hplus=4
```

Comment:
We don't use this anymore, so would you delete this and docker-compose.yaml files?